lib/BiochemistryAPI/utils.py

```diff
@@ -59,6 +59,12 @@ def dict_from_file(path, key="id", dialect="excel-tab"):
             alias_dict[tok].add(line[key])
         for tok in _tokenize_string(line.get("ec_number")):
             alias_dict[tok].add(line[key])
+        if line.get("aliases"):
```

Comment:
I guess scientists don't always do this and they're fine ... but personally I'd at least test this in a python shell with a bunch of inputs/outputs. I might even split this into a separate function and write some code testing inputs/outputs, since I am not a regex expert. Also, I personally would rename `_tokenize_string` with something more evocative/comprehensive like `_tokenize_retain_string` but YMMV

Reply:
Yea the formatting of the aliases in the first place is the problem, this will be fixed at the source